python/tvm/autotvm/tophub.py

```diff
@@ -0,0 +1,107 @@
+"""
+TopHub: Tensor Operator Hub
+To get the best performance, we typically need auto-tuning for the specific devices.
+TVM releases pre-tuned parameters in TopHub (https://github.com/uwsaml/tvm-distro)
+for some common networks and hardware targets.
+TVM will donwload these parameters for you when you create the target for the first time.
+"""
+import os
+import json
+
+from .task import DispatchContext
+from .record import ApplyHistoryBest
+from ..contrib.util import tempdir
+from ..contrib.download import download
+
+AUTOTVM_TOPHUB_ROOT_PATH = os.path.join(os.path.expanduser('~'), ".tvm", "tophub")
+
+def load_context(target, rootpath=AUTOTVM_TOPHUB_ROOT_PATH):
```

Comment:
it is still better to allow the with block

```python
with tophub.context(target):
   my code
```

Reply:
Is it possible to allow user also specify their customized location of tunning logs?